test/common/logging-methods.test.js

```diff
@@ -6,16 +6,34 @@ const logdown = require('../../src/node')
 // Tests
 //
 
+function beforeEach () {
```

Comment:
If you'll write the following:

```js
beaforeEach(() => {
  // ...
});
```

`jest` will call it automatically

Reply:
😮  Didn't know about that.
Can you make a PR fixing that?

Reply:
yep